tensorflow/core/kernels/non_max_suppression_op.cu.cc

```diff
@@ -28,26 +28,11 @@ limitations under the License.
 #include "tensorflow/core/util/gpu_launch_config.h"
 #include "tensorflow/stream_executor/stream_executor.h"
 
-#define TF_RETURN_IF_CUDA_ERROR(result)                   \
```

Comment:
Why are these macros removed in this PR?

Reply:
They are now in core/util/gpu_device_functions.h (as of PR #36641). The ones in non_max_suppression_op.cu.cc are duplicates.

Reply:
Then can we remove them in a separate PR?

Reply:
I suppose